dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -114,7 +115,9 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
         try {
             RunId response = clientWorkflowExecutionServiceApi.runWorkflow(workflowParams, languageType, workflowTypeVersion, TAGS,
                     "{}", workflowURL, workflowAttachment);
-            System.out.println("Launched WES run with id: " + response.toString());
+            String runID = response.getRunId();
+            out("Launched WES run with id: " + runID);
```

Comment:
I feel this phrasing could be better, but will defer to others. Maybe something like, which I'm not that wild about:

```
Your WES run has been submitted. Its id is <id>.
```

I don't like `run` and would rather call it a job, but looks like WES calls them runs too, so never mind that part.